tests/integrational/ContenttypeTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace tests\integrational;
```

Comment:
I had not noticed myself that the namespace in  [transfer/transfer TransferTestCase](https://github.com/transfer-framework/transfer/blob/1.0/tests/functional/TransferTestCase.php) does not have the proper case. I'll note that.